crates/dbs-boot/Cargo.toml

```diff
@@ -13,7 +13,7 @@ readme = "README.md"
 [dependencies]
 dbs-arch = { path = "../dbs-arch", version = "0.2.0" }
 kvm-bindings = { version = "0.6.0", features = ["fam-wrappers"] }
-kvm-ioctls = "0.11.0"
```

Comment:
Wouldn't it be better to put it in the first patch?

Reply:
I will move it into the first patch in next push.